openstf/model/train.py

```diff
@@ -73,29 +72,75 @@ def is_data_sufficient(data):
     return is_sufficient
 
 
-def train_model_for_specific_pj(
-    pj, context, retrain_young_models=False, compare_to_old=True
-):
+def train_model_pipeline(pj, context, retrain_young_models=False, compare_to_old=True):
     """Main function that controls the training process of one prediction job.
 
-    Here, on a high level, the input data is collected, features are applied and the
+    On a high level: first, the input data is collected, features are applied and the
     model is trained and stored. Additionally models for energy components are also
     trained if requested. Model is saved if better than the old model unless specified
     otherwise.
 
     Args:
         pj (dict): Prediction job
         context (openstf.task.utils.TaskContext): Task context for logging
-        retrain_young_models (bool): Retrain models younger then 7 days if True
+        retrain_young_models (bool): Retrain models younger then MAX_AGE_YOUNG_MODEL
+            days if True
         compare_to_old (bool): Compare new trained model to old trained model if True
```

Comment:
Maybe its more clear if you also explain the action here: Only save new model when its better than the old model. (or something similar)

Reply:
done